src/methods/fastify-plugin.ts

```diff
@@ -1,16 +1,16 @@
-import { FastifyInstance } from 'fastify';
+import { TypeBoxTypeProvider } from '@fastify/type-provider-typebox';
+import { FastifyInstance, FastifyRequest } from 'fastify';
 import fp from 'fastify-plugin';
 import { handlePubSubMessage } from '../common';
-import { PubSubConfig, PubSubRequest, PubSubRequestType } from '../types';
+import { PubSubConfig, PubSubRequest } from '../types';
 
-const pubSubFastifyPluginFn = async <Data, Context>(
+const pubSubFastifyPluginFn = async <Data = unknown>(
```

Comment:
I don't understand why you take `Context` as a generic when you just pass `request`. (or rather, before this PR it was never passed at all)